vispy/geometry/generation.py

```diff
@@ -89,6 +89,214 @@ def create_cube():
     return vertices, filled, outline
 
 
+def create_plane(width=1, height=1, width_segments=1, height_segments=1,
+                 direction='+z'):
+    """ Generate vertices & indices for a filled and outlined plane.
+
+    Parameters
+    ----------
+    width : float
+        Plane width.
+    height : float
+        Plane height.
+    width_segments : int
+        Plane segments count along the width.
+    height_segments : float
+        Plane segments count along the height.
+    direction: unicode
+        ``{'-x', '+x', '-y', '+y', '-z', '+z'}``
+        Direction the plane will be facing.
+
+    Returns
+    -------
+    vertices : array
+        Array of vertices suitable for use as a VertexBuffer.
+    faces : array
+        Indices to use to produce a filled plane.
+    outline : array
+        Indices to use to produce an outline of the plane.
+
+    References
+    ----------
+    [1] Cabello, R. (n.d.). PlaneBufferGeometry.js. Retrieved May 12, 2015,
+        from http://git.io/vU1Fh
```

Comment:
Well, I'm actually following it (I omitted the `..` by mistake, my bad): https://github.com/numpy/numpy/blob/master/numpy/core/fromnumeric.py#L2719

That being said Numpy doesn't seem to have a standard / common citation style. I used APA here (like in our API,  https://github.com/colour-science/colour/issues/135) which is quite common in the scientific community.

We do use a citation manager (Mendeley) and export everything from it for each release: https://github.com/colour-science/colour/blob/develop/BIBLIOGRAPHY.bib along a scrapping from all the API references: https://github.com/colour-science/colour/blob/develop/BIBLIOGRAPHY.rst

Now for the MIT thing, I don't remember it is something you find in citations but it can definitely be included.

Just let me know what do you want.